genome_designer/variants/tests/test_vcf_parser.py

```diff
@@ -48,6 +52,16 @@ def setUp(self):
         self.reference_genome = import_reference_genome_from_local_file(
                 self.project, 'ref_genome', TEST_FASTA, 'fasta')
 
+    def test_import_variant_set(self):
```

Comment:
Another way to write this is

    import_variant_set_from_vcf(
            self.reference_genome,
            'test_variant_set',  # variant_set_name
            TEST_VARIANTS_IMPORT_VCF  # variant_set_file
    )